### ISSUE TITLE
No configuration to modify KeepActivityPrivate setting

### ISSUE THREAD

**Issue by element-wise** [go-gitea/gitea#29202]
### Description

I am trying to disable public activity for a user with "public" visibility. There is an user option "KeepActivityPrivate" per #11353 that serves this purpose but there isn't a [service config](https://github.com/go-gitea/gitea/blob/main/modules/setting/service.go) in app.ini to disable public activity.

### Gitea Version

1.21.4

### Can you reproduce the bug on the Gitea demo site?

Yes

### Git Version

2.43.0

### Operating System

Linux

### How are you running Gitea?

Docker

### Database

SQLite

**Comment by techknowlogick** [comment]
Hi, thanks for opening this report. To clarify, you'd like an option in the config to disable to public activity portion of all users on the instance?

**Comment by element-wise** [comment]
@techknowlogick, that's correct. Correct me if I'm wrong: there already exists a user-level setting to disable his or her public activities from being visible via KeepActivityPrivate param (see #11353). I saw user profile template make use of this param (see [profile.tmpl#L14](https://github.com/go-gitea/gitea/blob/26b17537e651fe93ef9b64f961633cb4c0b8c2c3/templates/user/profile.tmpl#L14C1-L15C1)) but couldn't figure out how to modify it.
